mangaki/mangaki/tests/test_anidb.py

```diff
@@ -25,10 +25,21 @@ def setUp(self):
         # exist, or else foreign key constraints fail.
         Editor.objects.create(pk=1)
         # Studio.objects.create(pk=1)
-        self.anidb = client
+        self.anidb = AniDB('test_client', 1)
+        self.no_anidb = AniDB()
         self.search_fixture = self.read_fixture('search_sangatsu_no_lion.xml')
         self.anime_fixture = self.read_fixture('anidb/sangatsu_no_lion.xml')
 
+    def test_to_python_datetime(self):
```

Comment:
I usually prefer `doctests` for this kind of function as they're not attached to `AniDB` in particular, you can test exceptions also with [`doctests`](https://docs.python.org/3.5/library/doctest.html)